test/emqx_ocsp_cache_SUITE.erl

```diff
@@ -291,6 +307,12 @@ get_sni_fun(ListenerID) ->
     SSLOpts = proplists:get_value(ssl_options, Opts),
     proplists:get_value(sni_fun, SSLOpts).
 
+openssl_version() ->
+    Res0 = os:cmd("openssl version"),
```

Comment:
may be something like this?
```suggestion
    Res0 = string:trim(os:cmd("openssl version"), trailing),
```

here is what I get
```bash
1> Res0 = os:cmd("openssl version").
"LibreSSL 2.8.3\n"
2> [_, Res] = string:split(Res0, " ").
["LibreSSL","2.8.3\n"]
3> {match, [Version]} = re:run(Res, "^([^ ]+)", [{capture, first, list}]).
{match,["2.8.3\n"]}
```

on linux (where openssl is actually openssl) newline characters comes after the third token and you don't have this problem there
```bash
4> [_, Res] = string:split(Res0, " ").
["OpenSSL","1.1.1f  31 Mar 2020\n"]
5> {match, [Version]} = re:run(Res, "^([^ ]+)", [{capture, first, list}]).
{match,["1.1.1f"]}
```

Reply:
nvm, since we want to run openssl cmd, it's os:cmd to be used to check its version.